### ISSUE TITLE
Can't assing asset to asset

### ISSUE THREAD

**Issue by TonyTheDarkLord** [snazy2000/SnipeitPS#58]
Hello, I can't make Set-AssetOwner work assigning Asset to another asset. Every time I try I get this error, with every asset. It's a bad understanding or not working?

![Error](https://user-images.githubusercontent.com/7749082/89743403-4a42a100-da79-11ea-863d-d2a7ef3d240b.png)

Thanks in advance.

Regards,
Juan Pablo

**Comment by sheppyh** [comment]
I also have this problem. SnipeIt API is raising the error, but not sure if it's because of what SnipeitPS is passing @TonyTheDarkLord

**Comment by sheppyh** [comment]
@TonyTheDarkLord @snazy2000 I think the problem is here:
![image](https://user-images.githubusercontent.com/10708793/92062898-ac00dd00-ed91-11ea-9653-9de8b25d5a18.png)

```powershell
    $Values = @{
        "id" = $assigned_id
        "checkout_to_type" = $checkout_to_type
    }
```

Should be

```powershell    
    $Values = @{
        "id" = $id
        "checkout_to_type" = $checkout_to_type
    }
```

**Comment by dilpatel45** [comment]
Hey guys, i'm having the same issue. Is there a workaround or another solution to checkout asset to asset?

**Comment by Blakeykplc** [comment]
Hey All, i am experiencing this too when trying to assign assets to assets? any updates? or workarounds?

**Comment by PetriAsi** [comment]
@sheppyh  You suggestion fixed this. changed  "id" = $assigned_id" to  "id" = $id as you suggested.
This is now fixed in develop.

**Comment by PetriAsi** [comment]
Fixed in version v1.1.x, closing this now. Reopen if needed.
